android-sdk/src/androidTest/java/com/mobilecoin/lib/MnemonicTest.java

```diff
@@ -88,9 +92,28 @@ public void words_by_prefix() throws BadMnemonicException {
         assertArrayEquals(new String[]{"zebra", "zero", "zone", "zoo"}, words);
     }
 
+    @Test
+    public void all_english_words_are_correct()
+            throws NoSuchAlgorithmException, BadMnemonicException {
+        assertEquals(
+                "ffbc2f3228ee610ad011ff9d38a1fb8e49e23fb60601aa7605733abb0005b01e",
```

Comment:
Good idea. Is this value part of the BIP-39 spec somewhere?

Reply:
No, it was provided by Alan

Reply:
Or maybe it is, but I didn't see it in the test vectors